velox/functions/sparksql/aggregates/tests/BloomFilterAggAggregateTest.cpp

```diff
@@ -86,4 +87,31 @@ TEST_F(BloomFilterAggAggregateTest, nullBloomFilter) {
           vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expectedFake),
       "First argument of bloom_filter_agg cannot be null");
 }
+
+TEST_F(BloomFilterAggAggregateTest, config) {
+  auto vector = {makeRowVector({makeFlatVector<int64_t>(
+      100, [](vector_size_t row) { return row % 9; })})};
+  auto bloomFilter = getSerializedBloomFilter(100);
+  std::vector<RowVectorPtr> expected = {
+      makeRowVector({makeConstant<StringView>(StringView(bloomFilter), 1)})};
+
+  // This config will decide the bloom filter capacity, the expected value is
+  // the serialized bloom filter, it should be consistent.
+  testAggregations(
+      vector,
+      {},
+      {"bloom_filter_agg(c0)"},
+      expected,
+      {{core::QueryConfig::kSparkBloomFilterMaxNumBits, "1600"}});
+
+  exec::test::PlanBuilder builder(pool());
+  builder.values(vector)
+      .partialAggregation({}, {"bloom_filter_agg(c0)"})
+      .finalAggregation();
+  exec::test::AssertQueryBuilder queryBuilder(
+      builder.planNode(), duckDbQueryRunner_);
+  auto actual = queryBuilder.copyResults(pool());
+  EXPECT_FALSE(
```

Comment:
What does this do? Would you explain this test scenario? Perhaps, add a comment.

Reply:
For this comment https://github.com/facebookincubator/velox/pull/5575#discussion_r1261105884